modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -70,6 +70,10 @@ On Ubuntu/Debian/Mint systems:
 sudo apt install php-apcu
 ----
 
+On Ubuntu 22.04 with PHP 7.4 from ondrej/php PPA:
+
+sudo apt install php7.4-apcu
+
```

Comment:
Move this to [the https://github.com/owncloud/docs-server/pull/369/files#](https://github.com/owncloud/docs-server/pull/369/files#diff-cb5c0cea222e6d376fca4445f6c9b351ea8e79ff042e9f8045a5c8f644a892f0R73-R76)